src/drupal8/docs/customise/commands.md

```diff
@@ -0,0 +1,42 @@
+# Adding custom commands
+
+Adding commands can help with productivity and standardise tasks.  
+Information on writing commands can be found [here](https://github.com/my127/workspace/blob/0.1.x/docs/types/command.md)
+
+Here are some example that might be a good starting point.
+
+
+### Refresh script - `ws drupal-refresh`
+```yaml
+command('drupal-refresh'):
+  exec: |
+    #!bash(workspace:/)|@
+    ws composer install
+    ws drush -y cr
+    ws drush -y updb
+    ws frontend install
+```
+
+
+### Database import - `ws import-db path/to/db.sql`
+```yaml
+command('drush-import-db <alias> <filename>'):
```

Comment:
I think this can be made into the `ws db import <filename>` here: https://github.com/inviqa/harness-base-php/blob/0.6.x/src/_base/harness/config/commands.yml#L212-L217 - maybe an overridden version for the drupal8 harness?

Reply:
I thought I would leave this example in, just because the base import command doesn't account for multiple database set ups. Happy to change though.

Reply:
Tis okay :)